media/js/base/mozilla-client.js

```diff
@@ -27,8 +100,20 @@ if (typeof Mozilla === 'undefined') {
     };
 
     /**
+     * Get the user's Firefox major version number. 0 will be returned on non-Firefox browsers.
+     *
+     * @private
+     * @param  {String} ua - browser's user agent string, navigator.userAgent is used if not specified
+     * @return {Number} major version number
+     */
+    Client._getFirefoxMajorVersion = function (ua) {
+        return parseFloat(Client._getFirefoxVersion(ua));
```

Comment:
If we're relying on the expectation that the minor version number is never in the UA string for this to work, I'd feel more comfortable if we explicitly made this return only the major version, like the function name suggests.